### PR TITLE
[FM v0.6.0]--Add Doc release note

### PR DESCRIPTION
**Pull request by Huanli-Meng**
### Motivation

Add a doc release note for doc releases for FM v0.6.0

### CODE REVIEW

File: docs/releases/release-note-0-6-0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+id: release-note-0-6-0
+---
+
+Here are some highlights of this release. For a full list of updates available for Release v0.6.0, check out the [Function Mesh change log](https://github.com/streamnative/function-mesh/releases/tag/v0.6.0).
+
+## Support window functions
+
+A window function is a function that performs computation across a window. Java SDK provides access to a window context object that can be used by a window function. This context object provides a wide variety of information and functionality for a Pulsar window function.
+
+For details, see [window functions](functions/function-overview.md#window-functions).
```
Comment (freeznet) [inline docs/releases/release-note-0-6-0.md:13]:
```suggestion
Function Mesh adds `WindowConfig` to allow the user to pass Pulsar Window Function configurations and run the Window Function with Function Mesh. For details, see [window functions](functions/function-overview.md#window-functions).

```

**Review by freeznet — APPROVED** [review]

File: docs/releases/release-note-0-6-0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+id: release-note-0-6-0
+---
+
+Here are some highlights of this release. For a full list of updates available for Release v0.6.0, check out the [Function Mesh change log](https://github.com/streamnative/function-mesh/releases/tag/v0.6.0).
+
+## Support window functions
+
+A window function is a function that performs computation across a window. Java SDK provides access to a window context object that can be used by a window function. This context object provides a wide variety of information and functionality for a Pulsar window function.
+
+Function Mesh supports the `WindowConfig` option that allows users to pass Pulsar window function configurations and run a window function with Function Mesh. For details, see [window functions](functions/function-overview.md#window-functions).
```
Comment (teachwrite) [inline docs/releases/release-note-0-6-0.md:13]:
```suggestion
Function Mesh supports the `WindowConfig` option that allows you to pass Pulsar window function configurations and run a window function with Function Mesh. For details, see [window functions](functions/function-overview.md#window-functions).
```

File: docs/releases/release-note-0-6-0.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Support OAuth2 configurations
+
+Function Mesh supports the OAuth2 authentication options. If a Pulsar cluster uses OAuth2 authentication, users can directly configure the OAuth2 authentication using the `authConfig` fields in the Function, Source, and Sink CRDs. For details about OAuth2 authentication, see [authentication](/functions/function-crd.md#authentication).
```
Comment (teachwrite) [inline docs/releases/release-note-0-6-0.md:17]:
```suggestion
Function Mesh supports the OAuth2 authentication options. If a Pulsar cluster uses OAuth2 authentication, you can directly configure the OAuth2 authentication using the `authConfig` fields in the Function, Source, and Sink CRDs. For details about OAuth2 authentication, see [authentication](/functions/function-crd.md#authentication).
```

**Review by teachwrite — APPROVED** [review]
Changed "user" to "you" in two places, otherwise this LGTM.

**Review by freeznet — APPROVED** [review]

**Pull request merged**